packages/tab-bar/tab-bar.ts

```diff
@@ -108,7 +108,11 @@ export class MdcTabBar implements AfterContentInit, OnDestroy {
     getScrollPosition: () => this.tabScroller.getScrollPosition(),
     getScrollContentWidth: () => this.tabScroller.getScrollContentWidth(),
     getOffsetWidth: () => this._getHostElement().offsetWidth,
-    isRTL: () => window.getComputedStyle(this._getHostElement()).getPropertyValue('direction') === 'rtl',
+    isRTL: () => {
```

Comment:
```ts
isRtl: () => isBrowser() ? window.getComputedStyle(this._getHostElement()).getPropertyValue('direction') : false,
```

Reply:
Are we changing function name from isRTL to isRtl?

Reply:
Good catch! Should be `isRTL`, and returns boolean.

```ts
isRTL: () => isBrowser() ? window.getComputedStyle(this._getHostElement()).getPropertyValue('direction') === 'rtl' : false,
```

Reply:
Fixed